js_modules/dagster-ui/packages/ui-core/src/assets/CurrentRunsBanner.tsx

```diff
@@ -50,12 +50,12 @@ export const CurrentRunsBanner = ({
                   {inProgressRunIds.length && unstartedRunIds.length ? ' ' : ''}
                   {unstartedRunIds.length > 1 && (
                     <>
-                      Runs <RunIdLinks ids={unstartedRunIds} /> that target this asset are queued.
+                      Runs <RunIdLinks ids={unstartedRunIds} /> are queued and target this asset.
                     </>
                   )}
                   {unstartedRunIds.length === 1 && (
                     <>
-                      Run <RunIdLinks ids={unstartedRunIds} /> that targets this asset are queued.
+                      Run <RunIdLinks ids={unstartedRunIds} /> is queued and targets this asset.
                     </>
```

Comment:
seems like a good opportunity to use the `ifPlural` util, tho honestly I don't like the API that much, maybe it should take named parameter args 😅

Reply:
Ahh interesting I forgot we had this! I tried to swap it but this one is a bit of a pain because the text before and after the RunIdLinks both change (Run => Runs and is => are). I think this format is better than using isPlural twice, but we should def try to advance that util!